registration-default.properties

```diff
@@ -403,31 +454,70 @@ mosip.kernel.uin.length.repeating-limit=${mosip.kernel.uin.length.repeating-limi
 mosip.kernel.uin.length.conjugative-even-digits-limit=${mosip.kernel.uin.length.conjugative-even-digits-limit}
 mosip.kernel.uin.length.reverse-digits-limit=${mosip.kernel.uin.length.reverse-digits-limit}
 mosip.kernel.uin.length.digits-limit=${mosip.kernel.uin.length.digits-limit}
-
-
 mosip.kernel.vid.restricted-numbers=${mosip.kernel.vid.restricted-numbers}
 mosip.kernel.vid.not-start-with=${mosip.kernel.vid.not-start-with}
 mosip.kernel.vid.length.repeating-limit=${mosip.kernel.vid.length.repeating-limit}
 mosip.kernel.vid.length.repeating-block-limit=${mosip.kernel.vid.length.repeating-block-limit}
 mosip.kernel.vid.length.sequence-limit=${mosip.kernel.vid.length.sequence-limit}
 mosip.kernel.vid.length=${mosip.kernel.vid.length}
-
-mosip.kernel.xsdstorage-uri=LOCAL
-mosip.kernel.xsdfile=LOCAL
-
 mosip.kernel.registrationcenterid.length=${mosip.kernel.registrationcenterid.length}
 mosip.kernel.machineid.length=${mosip.kernel.machineid.length}
-
 ## RID
 mosip.kernel.rid.length=${mosip.kernel.rid.length}
 mosip.kernel.rid.timestamp-length=${mosip.kernel.rid.timestamp-length}
 mosip.kernel.rid.sequence-length=${mosip.kernel.rid.sequence-length}
-
 ## Virus scanner
 # Here we specify the Kubernetes service name if clamav runs inside cluster
 mosip.kernel.virus-scanner.host=${mosip.kernel.virus-scanner.host}
 mosip.kernel.virus-scanner.port=${mosip.kernel.virus-scanner.port}
-
 mosip.kernel.otp.expiry-time=${mosip.kernel.otp.expiry-time}
-
 ## end of global properties
+
+## backward compatibility fields
+mosip.registration.uin.update.configured.fields=test	
+mosip.primary-language=${mosip.primary-language}	
+mosip.secondary-language=${mosip.secondary-language}	
+mosip.registration.ideal_time=900	
+mosip.kernel.applicant.type.age.limit=0	
+mosip.registration.max_age=200
+
+# Parameters required for Backward Compatibility
+# These parameters needed for supporting previous versions of registration clients in case of upgraded env.
+
+mosip.registration.mdm.hostProtocol=http
+mosip.mdm.enabled=TRUE
+mosip.registration.ui_sync_data=y
+mosip.registration.Login_Credentials_Sync.frequency=190
+mosip.registration.consent_fra="Je comprends que les données recueillies à mon sujet lors de la enregistrement par ladite autorité comprennent mon -  <br> &#x2022 nom <br>&#x2022 Date de naissance <br>&#x2022 genre <br>&#x2022 adresse <br>&#x2022 coordonnées <br>&#x2022 Documents <br>Je comprends également que ces informations seront stockées et traitées dans le but de vérifier mon identité afin d'accéder à divers services, ou de se conformer à une obligation légale. Je donne mon consentement pour la collecte de ces données à cette fin."
```

Comment:
Many unused properties are part of this file. This cleanup was done during 1.2.0 release.
Pls refer this https://github.com/mosip/mosip-config/blob/develop-v3/registration-default.properties

Reply:
updated as per develop-v3